.github/workflows/workflow.yml

```diff
@@ -19,6 +19,8 @@ jobs:
         with:
           python-version: 3.6
       - name: Install dependencies (ubuntu-latest)
+        env:
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: 
```

Comment:
https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/